meteomatics/api.py

```diff
@@ -727,14 +733,22 @@ def query_polygon(latlon_tuple_lists, startdate, enddate, interval, parameters,
         coordinates_polygon_list.append(coordinates_polygon)
 
     if len(coordinates_polygon_list) > 1:
-        coordinates = operator.join(coordinates_polygon_list)
+        if operator is not None:
+            coordinates = operator.join(coordinates_polygon_list)
+            coordinates = coordinates + ':' + aggregation[0]
+        else:
+            coordinates_polygon_list_aggregator_included = []
+            for i, coordinates_polygon in enumerate(coordinates_polygon_list):
+                coordinates_polygon = coordinates_polygon + ':' + aggregation[i]
+                coordinates_polygon_list_aggregator_included.append(coordinates_polygon)
+            coordinates = '+'.join(coordinates_polygon_list_aggregator_included)
+
     else:
-        coordinates = coordinates_polygon_list
+        coordinates = coordinates_polygon_list[0] + ':' + aggregation[0]
```

Comment:
Sicher, dass die nicht leer sind?

Reply:
Ja das passt :) ... es ist ja eine Liste von Listen und wenn nur eine Liste in der Liste ist, wählt man die aus: [[(45.1, 8.2), (45.2, 8.0), (46.2, 7.5)]]